doc/ref_arch/kubernetes/chapters/chapter05.md

```diff
@@ -68,30 +94,159 @@ between tenants
+ports which makes it easy to identify the clusters and attack them. Hence, it is
+highly recommended to configure authentication and authorization on the cluster
+and cluster nodes.
+
+Here is an overview of the default ports used in Kubernetes. Make sure that your
+network blocks access to ports and consider limiting access to the Kubernetes
+API server except from trusted networks.
+
+**Master node(s):**
+
+| Protocol | Port Range | Purpose                 |
+| -------- | ---------- | ----------------------- |
+| TCP      | 6443-      | Kubernetes API Server   |
+| TCP      | 2379-2380  | etcd server client API  |
+| TCP      | 10250      | Kubelet API             |
+| TCP      | 10251      | kube-scheduler          |
+| TCP      | 10252      | kube-controller-manager |
+| TCP      | 10255      | Read-Only Kubelet API   |
+
+**Worker nodes:**
+
+| Protocol | Port Range  | Purpose               |
+| -------- | ----------- | --------------------- |
+| TCP      | 10250       | Kubelet API           |
+| TCP      | 10255       | Read-Only Kubelet API |
+| TCP      | 30000-32767 | NodePort Services     |
+
+### 5.4.2 Controlling access to the Kubernetes API
+The Kubernetes platform is controlled using API requests and as such is the
+first line of defence against attackers. Controlling who has access and what
+actions they are allowed to perform is the primary concern
+    
+### 5.4.3 Use Transport Layer Security
+Communication in the cluster between services should be handled using TLS,
+encrypting all traffic by default. This, however, is often overlooked with the
+thought being that the cluster is secure and there is no need to provide
+encryption in transit within the cluster.
+
+Advances in network technology, such as the service mesh, have led to the
+creation of products like LinkerD and Istio which can enable TLS by default
+while providing extra telemetry information on transactions between services.
+
+Kubernetes expects that all API communication in the cluster is encrypted by
+default with TLS, and the majority of installation methods will allow the
+necessary certificates to be created and distributed to the cluster components.
+Note that some components and installation methods may enable local ports over
+HTTP and administrators should familiarize themselves with the settings of each
+component to identify potentially unsecured traffic.
+
+### 5.4.4 API Authentication, API Authorization
 Secure all connections to a Kubernetes Cluster. Adopt the following security
 authentication mechanisms:
- - Configure user roles and access levels to provide segregation of duties (RBAC)
- - Use multi-factor authentication for all administrative access
- - Use token-based or certificate-based service and session authentication mechanisms
- - Integrated with existing identity management platforms e.g SAML, AD, etc. for
- access control
+* Configure user roles and access levels to provide segregation of duties (RBAC)
```

Comment:
What is the change in the following bullets?

Reply:
Not much, the markdown link must've updated the format.